src/components/Filters/SearchGridFilter.vue

```diff
@@ -144,15 +144,19 @@ export default {
 
 .search-filters {
   display: none;
-  height: auto;
   top: 0;
   position: sticky;
+  height: auto;
 
   label {
     color: #333333;
   }
 
   &__visible {
+    @media (min-width: 768px) {
```

Comment:
Could you use Bulma's [device](https://github.com/jgthms/bulma/blob/master/sass/utilities/mixins.sass#L81) mixins here? In this way, we will have consistency everywhere! you rever to this PR #881 for some help!